cpp/inline/games/stochastic_nim/PerfectPlayer.inl

```diff
@@ -0,0 +1,58 @@
+#include <games/stochastic_nim/PerfectPlayer.hpp>
+
+namespace stochastic_nim {
+
+inline PerfectPlayer::ActionResponse PerfectPlayer::get_action_response(
+    const State& state, const ActionMask& valid_actions) {
+  util::release_assert(state.current_mode == kPlayerMode,
+                       "PerfectPlayer does not support chance mode");
```

Comment:
In my opinion, this string adds no value. The bool condition is clear enough, it doesn't need further-expounding.

I would only add helpful string messages like this for errors that result from **runtime** human error (like passing bad values in the command line). If it's to catch **coding** errors, I only add formatted messages to display variable values that would be helpful for debugging.